Gemfile

```diff
@@ -88,4 +88,6 @@ group :development do
   gem 'mailcatcher', '~> 0.5.12'
   gem 'quiet_assets', '~> 1.1.0'
   gem 'rdoc', '~> 3.12.2'
+  gem 'better_errors'
```

Comment:
- Should add a [pessimistic version constraint](http://guides.rubygems.org/patterns/#pessimistic-version-constraint) to all gems
- better_errors has [dropped 1.9 support](https://github.com/charliesome/better_errors/pull/237) – should add a `:platforms => [:ruby_20]` constraint. Unfortunately `:ruby_20` is the latest version of ruby [our version of bundler](https://github.com/bundler/bundler/blob/master/CHANGELOG.md#130pre3-dec-21-2012) supports.